docs/en/connector-v2/sink/Greenplum.md

```diff
@@ -0,0 +1,17 @@
+# Greenplum
+
+> Greenplum sink connector
```

Comment:
Need to indicate whether two-phase commit is supported, which is related to exactly-once semantics.
How to configure `xa_data_source_class_name` if you perform a two-phase commit.
It's better to add a job configuration example.

Reply:
XA transaction is not yet supported in Greenplum Database

Reply:
https://docs.vmware.com/en/VMware-Tanzu-Greenplum/6/greenplum-database/GUID-ref_guide-feature_summary.html